packages/reactotron-core-server/src/reactotron-core-server.ts

```diff
@@ -98,9 +103,11 @@ export default class Server {
 
     // register events
     this.wss.on("connection", (socket, request) => {
+      const thisConnectionId = this.connectionId++
+
       // a wild client appears
       const partialConnection = {
-        id: (socket as any).id, // issue
```

Comment:
@skellock I suspect `socketio` gave us an id but `ws` does not. So we make a incrementing one similar to message ids.

Reply:
Oops.